web/src/ui/timeline/content/PolicyRuleBody.tsx

```diff
@@ -0,0 +1,28 @@
+import { PolicyRuleContent } from "@/api/types"
+import EventContentProps from "./props.ts"
+
+const BanPolicyBody = ({ event, sender }: EventContentProps) => {
+	const content = event.content as PolicyRuleContent
+	const prevContent = event.unsigned.prev_content as PolicyRuleContent | undefined
+
+	if (prevContent !== undefined) {
+		// all fields for content are missing (this is against spec?) so we need to use the prev event's content
+		if (content.entity === undefined) {
```

Comment:
There's no reason to check `=== undefined` specifically, `if (content.entity)` is sufficient